### ISSUE TITLE
Explorer Windows opens in the background instead of the foreground

### ISSUE THREAD

**Issue by Iqigai** [spyder-ide/spyder#15066]
### What steps will reproduce the problem?

Spyder 4.2.5 running on Windows 10
From the file panel, select a file.
Select and click on the "Show in Folder" menu item from the right click contextual menu.
The explorer opens in the background, instead of the foreground so it feels that the command did not work and you tend to try again before realizing that the Explorer Windows were openned in the background.
This feature could be enhanced by opening the window in the foreground, on top of the Spyder window.
Thank you!

## Versions

* Spyder version: 4.2.5 
* Python version: 3.9.2
* Qt version: 5.12.10
* PyQt5 version: 5.12.3
* Operating System: Windows 10

### Dependencies

```

# Mandatory:
atomicwrites >=1.2.0            :  1.4.0 (OK)
chardet >=2.0.0                 :  4.0.0 (OK)
cloudpickle >=0.5.0             :  1.6.0 (OK)
diff_match_patch >=20181111     :  20200713 (OK)
intervaltree >=3.0.2            :  3.1.0 (OK)
IPython >=7.6.0                 :  7.20.0 (OK)
jedi =0.17.2                    :  0.17.2 (OK)
jsonschema >=3.2.0              :  3.2.0 (OK)
keyring >=17.0.0                :  23.0.1 (OK)
nbconvert >=4.0                 :  6.0.7 (OK)
numpydoc >=0.6.0                :  1.1.0 (OK)
paramiko >=2.4.0                :  2.7.2 (OK)
parso =0.7.0                    :  0.7.0 (OK)
pexpect >=4.4.0                 :  4.8.0 (OK)
pickleshare >=0.4               :  0.7.5 (OK)
psutil >=5.3                    :  5.8.0 (OK)
pygments >=2.0                  :  2.7.4 (OK)
pylint >=1.0                    :  2.7.2 (OK)
pyls >=0.36.2;<1.0.0            :  0.36.2 (OK)
pyls_black >=0.4.6              :  0.4.6 (OK)
pyls_spyder >=0.3.2             :  0.3.2 (OK)
qdarkstyle >=2.8;<3.0           :  2.8.1 (OK)
qtawesome >=0.5.7               :  1.0.2 (OK)
qtconsole >=5.0.3               :  5.0.3 (OK)
qtpy >=1.5.0                    :  1.9.0 (OK)
setuptools >=39.0.0             :  49.2.1 (OK)
sphinx >=0.6.6                  :  3.5.3 (OK)
spyder_kernels >=1.10.2;<1.11.0 :  1.10.2 (OK)
textdistance >=4.2.0            :  4.2.1 (OK)
three_merge >=0.1.1             :  0.1.1 (OK)
watchdog >=0.10.3;<2.0.0        :  1.0.2 (OK)
zmq >=17                        :  22.0.2 (OK)

# Optional:
cython >=0.21                   :  0.29.21 (OK)
matplotlib >=2.0.0              :  3.3.4 (OK)
numpy >=1.7                     :  1.19.5 (OK)
pandas >=1.1.1                  :  1.2.3 (OK)
scipy >=0.17.0                  :  1.6.0 (OK)
sympy >=0.7.3                   :  None (NOK)
```

**Comment by Iqigai** [comment]
Hi @steff456,
Here are the requested screen captures. I have reduced the Spyder window so that you can see what happens better. Capture1.jpg is the selection of the file I want to see with right click and selecting the "Show in folder"
Capture2.jpg shows that the the Explorer window was opened "behind" Spyder, we can see it only because I have reduced the size of the Spyder window to occupy a smaller portion of the screen. I think that the Explorer window should be brought to the front and should open on top of Spyder, that would be more user friendly and in a way more logical.
Thank you!
![Capture1](https://user-images.githubusercontent.com/80190909/113619108-9d511000-9669-11eb-9c1f-d33c31ea6218.JPG)
![Capture2](https://user-images.githubusercontent.com/80190909/113619097-9c1fe300-9669-11eb-9b31-efb2c11c4624.JPG)

**Comment by ccordoba12** [comment]
@Iqigai, thanks for the graphical explanation. Now we understand what's happening.

@dalthviz, could you test this on Windows to see if you get the same result?

**Comment by steff456** [comment]
@Iqigai,

Did you set the icon theme of Spyder 2 in your preferences? Or your icons have always look like the ones you are having in the screenshot you post?

**Comment by dalthviz** [comment]
@Iqigai @ccordoba12 @steff456 I was able to reproduce this:


![explorer](https://user-images.githubusercontent.com/16781833/113757737-40119900-9713-11eb-954f-455507ce0edb.gif)

**Comment by Iqigai** [comment]
@steff456 
Yes I did set my icon preferences to Spyder 2, given that the Spyder 3 icons' rendering is not to my taste
